docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -18,45 +18,48 @@
-* The new Trigger instance is registered inside Airflow, and picked up by a *triggerer* process
-* The trigger is run until it fires, at which point its source task is re-scheduled
-* The scheduler queues the task to resume on a worker node
+An overview of how this process works:
 
-Using deferrable operators as a DAG author is almost transparent; writing them, however, takes a bit more work.
+* A task instance (running operator) reaches a point where it has to wait for other operations or conditions, and defers itself with a trigger tied to an event to resume it. This frees up the worker to run something else.
+* The new trigger instance is registered by Airflow, and picked up by a triggerer process.
+* The trigger runs until it fires, at which point its source task is re-scheduled by the scheduler.
+* The scheduler queues the task to resume on a worker node.
 
+You can either use pre-written deferrable operators as a DAG author or write your own. Writing them, however, requires that they meet certain design criteria.
 
 Using Deferrable Operators
 --------------------------
 
-If all you wish to do is use pre-written Deferrable Operators (such as ``TimeSensorAsync``, which comes with Airflow), then there are only two steps you need:
+If you want to use pre-written deferrable operators that come with Airflow, such as ``TimeSensorAsync``, then you only need to complete two steps:
 
-* Ensure your Airflow installation is running at least one ``triggerer`` process, as well as the normal ``scheduler``
+* Ensure your Airflow installation runs at least one ``triggerer`` process, as well as the normal ``scheduler``
 * Use deferrable operators/sensors in your DAGs
 
-That's it; everything else will be automatically handled for you. If you're upgrading existing DAGs, we even provide some API-compatible sensor variants (e.g. ``TimeSensorAsync`` for ``TimeSensor``) that you can swap into your DAG with no other changes required.
+Airflow automatically handles and implements the deferral processes for you.
 
-Note that you cannot yet use the deferral ability from inside custom PythonOperator/TaskFlow Python functions; it is only available to traditional, class-based Operators at the moment.
+If you're upgrading existing DAGs to use deferrable operators, Airflow contains API-compatible sensor variants, like ``TimeSensorAsync`` for ``TimeSensor``. Add these variants into your DAG to use deferrable operators with no other changes required.
+
+Note that you can't use the deferral ability from inside custom PythonOperator or TaskFlow Python functions. Deferral is only available to traditional, class-based operators.
 
 .. _deferring/writing:
 
 Writing Deferrable Operators
 ----------------------------
 
-Writing a deferrable operator takes a bit more work. There are some main points to consider:
+Writing a deferrable operator requires more configuration than updating your DAGs with pre-written operators. There are some main points to consider:
```

Comment:
```suggestion
When writing a deferrable operators these are the main points to consider:
```

If you are worried about carryover context from the last section, let's do this instead. Writing operators will naturally be more complex than using pre-written ones - I see no value in being more verbose here.